MarkdownKit/Sources/Common/Elements/Italic/MarkdownItalic.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 open class MarkdownItalic: MarkdownCommonElement {
   
-  fileprivate static let regex = "(.?|^)(\\*|_)(?=\\S)(.+?)(?<![\\*_\\s])(\\2)"
```

Comment:
This regex was implemented in https://github.com/bmoliveira/MarkdownKit/commit/761a30e8d5db9384d7bbaa9a5b83e6a7e8fd5443 to "fix" when the italic syntax is [combined with bold and/or other characters](https://github.com/bmoliveira/MarkdownKit/issues/97), but it was capturing the preceding character (e.g., when capturing `_italic_`, it would capture ` _italic_` with the space before. I reverted it as it doesn't make much sense for our use case, and it still works with bold text.